modules/github-repository-webhook/variables.tf

```
@@ -38,3 +38,19 @@ variable "webhook_secret" {
   type        = string
   default     = ""
 }
+
+variable "webhook_insecure_ssl" {
+  description = "Disable verify SSL certificates when delivering payloads"
+  type        = bool
+  default     = false
+}
+
+variable "webhook_content_type" {
+  description = "Webhook Content Type"
+  type        = string
+  default     = "json"
```

Shouldn't this be left as the original default?

```suggestion
  default     = "application/json"
```

---

@nitrocode Previously @dynamike referred me here [Github Webhook accepted vlaues](https://registry.terraform.io/providers/integrations/github/latest/docs/resources/repository_webhook#content_type)

I have tested both ways, with "application/json" and with "json" and both work without issue. The reason I decided to keep with "json" over the previous default was due to the documentation stating the proper accepted inputs even though both work and are acceptable.

---

ah ok I see https://github.com/terraform-aws-modules/terraform-aws-atlantis/pull/312#discussion_r996105398

Thank you.